exotica_core/include/exotica_core/problems/dynamic_time_indexed_shooting_problem.h

```diff
@@ -64,6 +67,14 @@ class DynamicTimeIndexedShootingProblem : public PlanningProblem, public Instant
     Eigen::MatrixXd get_Q(int t) const;               ///< Returns the precision matrix at time step t
     void set_Q(Eigen::MatrixXdRefConst Q_in, int t);  ///< Sets the precision matrix for time step t
 
+    Eigen::MatrixXd get_Qf() const;             ///< Returns the cost weight matrix at time N
+    void set_Qf(Eigen::MatrixXdRefConst Q_in);  ///< Sets the cost weight matrix for time N
```

Comment:
I realised that `get_Q(-1)` (or T-1) is the same as Qf.

Reply:
What's the issue here?